src/superface-test.test.ts

```diff
@@ -363,6 +368,99 @@ describe('SuperfaceTest', () => {
           ]
         );
       });
+
+      describe('and there already exists old traffic', () => {
```

Comment:
and => when?

Reply:
I changed it to `and when old traffic already exists`